### ISSUE TITLE
"My Profile" showing duplicate cards from yesterday that I did not actually earn.

### ISSUE THREAD

**Issue by jthiesen** [Razviar/mtga-pro-tracker#418]
**Describe the bug**
"My Arena" > "My Profile" showing duplicate cards from yesterday on today. I played a bit today, but gained to new cards, so pretty sure it is wrong that some of the same cards are showing again today as can be seen here:

https://mtgarena.pro/u/Zarraxia

I noticed that the tracker was in need of an update, so I auto updated. Then I checked the site to see my progress. That's when I noticed the duplicates. Due to knowing that I only earned gold and no new cards today, I'm pretty sure there is some sort of bug here.

**To Reproduce**
I have not tried, but it should still be visible on my profile through the link above.

**Expected behavior**
I was expecting to only see gold earned, but no new cards.

**Screenshots**
![image](https://user-images.githubusercontent.com/759016/58439515-bd5e3280-80d4-11e9-89f4-eca44fb8c0de.png)

**Additional context**
Here is my log file in case you need it for troubleshooting. Hopefully I found the correct file :)
[output_log.zip](https://github.com/Razviar/mtga-pro-tracker/files/3224970/output_log.zip)

**Comment by Aquaman88** [comment]
Thank you ! We will investigate your issue

**Comment by jthiesen** [comment]
Thanks, it seems to be continuing to behave a bit weird. Here's a screenshot of today and yesterday.

![image](https://user-images.githubusercontent.com/759016/58586951-861a8d80-825c-11e9-9cb9-f516f23ac42c.png)

Notice how Emmara showed up for a 3rd time on the 28th and on the 29th all the cards I earned are listed as if I earned them twice from the booster. I only got one of each. The totals seem correct as I have verified them against my in-game collection.

I have looked at the page on the dates below these and have not found any anything that looks incorrect on those, so the problem seems to have started on the 26th.

Hope this helps with troubleshooting. If you need any more info, just ask.

**Comment by jthiesen** [comment]
A bit more strangeness on the next 2 days. 

![image](https://user-images.githubusercontent.com/759016/58695412-a13ce880-8395-11e9-8cc2-f327871ead6b.png)

I created a new deck through crafting on May 30th, so the cards are correct, but not the amounts. I did not craft 12 of any one card :)

I believe it looked more correct when I looked at the page yesterday, but the amounts had changed when I checked this morning.
